eox_nelp/pearson_vue/pipeline.py

```diff
@@ -16,10 +18,43 @@
 from eox_nelp.edxapp_wrapper.student import anonymous_id_for_user
 from eox_nelp.pearson_vue.constants import PAYLOAD_CDD, PAYLOAD_EAD, PAYLOAD_PING_DATABASE
 from eox_nelp.pearson_vue.utils import update_xml_with_dict
+from eox_nelp.signals.utils import get_completed_and_graded
+
+logger = logging.getLogger(__name__)
 
 User = get_user_model()
 
 
+def terminate_not_full_completion_cases(user_id, course_id, **kwargs):
```

Comment:
ChatGPT suggest that this name `handle_course_completion_status` describes better the functionality

Reply:
302e8de11d0229eed849a0f45998648ce089ef3b